web/src/HUD.tsx

```diff
@@ -173,7 +176,7 @@ class HUD extends Component<HudProps, HudState> {
       )
     }
 
-    let tabNavRoute = (t: ResourceView, props: RouteComponentProps<any>) => {
+    let tabNavRoute = (t: ResourceView, sailUrl: string, props: RouteComponentProps<any>) => {
```

Comment:
you don't need the whole do-si-do of making this an argument and binding it? it's already available in the lexical closure.

Reply:
shoot you're right!